pyxem/tests/signals/test_electron_diffraction2d.py

```diff
@@ -53,14 +53,14 @@ def test_center_direct_beam_xc_return_shifts(self, diffraction_pattern):
             radius_finish=3,
             return_shifts=True,
         )
-        ans = np.array([[-0.45, -0.45], [0.57, 0.57], [-0.45, -0.45], [0.52, 0.52]])
+        ans = np.array([[[0.45, 0.45], [-0.57, -0.57]], [[0.45, 0.45], [-0.52, -0.52]]])
```

Comment:
when did these signs flip? Did it get documented?

Reply:
This was a consequence of switching the `get_direct_beam_position` output from `[y, x]` to `[x, y]`. Instead of doing major changes to the `center_direct_beam` at the time, the signal was simply inverted to the original `[y, x]` order inside `center_direct_beam`. Thus, the `return_shifts` returned "wrong" order of the shifts.

The changes in the unit test here fixes that.

Reply:
Yeah, sorry it's even in the CHANGELOG, I must have been asleep at the wheel when I wrote this comment